src/etc/inc/dyndns.class

```diff
@@ -1543,6 +1767,78 @@
 						$this->_debug($data);
 					}
 					break;
+				case 'dreamhost':
+				case 'dreamhost-v6':
+					$result = json_decode($data,true);
+					if ($this->_dnsVerboseLog) {
+						log_error(sprintf(gettext('_checkStatus() results: %1$s'), $data));
+					}
+					switch ($result['data']) {
+						case 'success':
+						case 'record_added':
+						case 'record_removed':
+							$status = $status_intro . $success_str . gettext("IP Address Changed Successfully!") . " (" . $this->_dnsIP . ")";
+							$successful_update = true;
+							break;
+						case 'no_record':
+							$status = $status_intro . $error_str . gettext("No record exists.");
+							break;
+						case 'no_type':
+							$status = $status_intro . $error_str . gettext("No type exists.");
+							break;
+						case 'no_value':
+							$status = $status_intro . $error_str . gettext("No value exists.");
+							break;
+						case 'no_such_record ':
```

Comment:
I have grouped the no_*/no_such_* values but there is a distinct difference between a record that does not exist and one that is invalid.